package/Config.in

```diff
@@ -1,6 +1,7 @@
 menu "Target packages"
 
 	source "package/busybox/Config.in"
+	source "package/duo-pinmux/Config.in"
```

Comment:
I think it could be better if we could move this line to somewhere after the line 456.

https://github.com/gtxzsxxk/duo-buildroot/blob/bde00fc82e7b4de3f6bf23797cff3f5544d19df0/package/Config.in#L456C1-L456C1

```
menu "Hardware handling"
menu "Firmware"
	source "package/am33x-cm3/Config.in"
	source "package/armbian-firmware/Config.in"
	source "package/b43-firmware/Config.in"
...
	source "package/raspi-gpio/Config.in"
	source "package/rdma-core/Config.in"
	source "package/read-edid/Config.in"
	source "package/rng-tools/Config.in"
```

Duo pinmux (temporary name) is a gpio tool. And I find the gpio tool for raspberry is listed in the menu `Hardware Handling`. So we'd better put our line into this sub-menu too. And please ensure that we follow the alphabet order when inserting our line. If I am not making myself clear, please directly ask me.